pkg/kubectl/cmd/drain_test.go

```diff
@@ -312,29 +304,13 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
-	missing_ds := extensions.DaemonSet{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:              "missing-ds",
-			Namespace:         "default",
-			CreationTimestamp: metav1.Time{Time: time.Now()},
-			SelfLink:          testapi.Default.SelfLink("daemonsets", "missing-ds"),
-		},
-		Spec: extensions.DaemonSetSpec{
-			Selector: &metav1.LabelSelector{MatchLabels: labels},
-		},
-	}
-
-	missing_ds_anno := make(map[string]string)
-	missing_ds_anno[api.CreatedByAnnotation] = refJson(t, &missing_ds)
-
 	orphaned_ds_pod := corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:              "bar",
 			Namespace:         "default",
 			CreationTimestamp: metav1.Time{Time: time.Now()},
 			Labels:            labels,
 			SelfLink:          testapi.Default.SelfLink("pods", "bar"),
-			Annotations:       missing_ds_anno,
 			OwnerReferences: []metav1.OwnerReference{
```

Comment:
This test setup isn't correct. An "orphaned" pod does not have any controller (controllerRef == nil).

Reply:
I am aware of it, but i didn't change it as it is not relevant to created-by annotation.

Reply:
It's actually related, because this controllerRef was mistakenly added here when replacing created-by with controllerRef https://github.com/kubernetes/kubernetes/commit/d76b08d15480cf1681239c919376d5672b904951#diff-db0998ca74ee87d106ee23d584f337a5R399

Reply:
I verified it was me who added the owner references. I did not know about orphaning at that moment (it was my first PR for k8s main repo). Thanks for catching it!